msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md

```diff
@@ -204,6 +204,35 @@ The following table provides the response codes:
 | **501** | API isn't supported in the current context.|
 | **1000** | App doesn't have permissions to allow share to stage.|
 
+## Get all app content in meeting side panel
+
+The `sharing.history.getContent` API enables to open content in side panel. It requests for all the content shared in the meeting stage and to display in side panel and share to stage.
```

Comment:
`sharing.history.getContent` does not directly enable to share content in side panel. 

`sharing.history.getContent` enables developers to fetch content that was shared in a meeting. This content can be hydrated into the app's side panel when side panel is opened.